### DISCUSSION TITLE
2007/06/14/%E5%BC%BA%E8%BF%AB%E7%A5%9E%E7%BB%8F%E7%97%87

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#165] [Announcements]
# 2007/06/14/%E5%BC%BA%E8%BF%AB%E7%A5%9E%E7%BB%8F%E7%97%87

生活中的神经官能强迫行为还是挺常见的，比如说反复洗手，反复查看煤气关没关等等。一般人有这种行为，都算不上病。只有有非常严重的症状表现，才算是“病”了。前些天，在电视上看到了一些病例，比如一个患者：

https://blog.qizhen.xyz/2007/06/14/%E5%BC%BA%E8%BF%AB%E7%A5%9E%E7%BB%8F%E7%97%87

**Comment by ruanqizhen** [comment]
之前博客下的留言：

miranda
2007年08月17日 14:58 编辑
呵呵，我也经常这样 

回复
PhantomKid
2007年07月20日 14:56 编辑
 从宿舍到教师的路上有一个转弯，每次走到这里都必须走出一个90度直角转弯。某天走到这里溜号了，走过去突然觉得刚才的转弯不够标准，于是再回去重新转个90度的弯……
哈哈 这个太搞笑了 简直有点像mr.bean的风格了

回复
Yaming
2007年06月15日 17:33 编辑
一直担心自己有强迫症，看了你举的例子，感觉好多了，至少不属于重度的
我走路不喜欢碰到地上砖块的拼缝。。。不知道为啥？

回复
竹贤
2007年06月15日 08:22 编辑
你夫人每天用电话指挥你查看煤气、水、门等是否关闭，到并非就是强迫症。对你而言，也许是知道你的大大咧咧，怕你丢三拉四，带来麻烦；对她而言，则明显是一种对新婚爱人的关心。
养成一个随时关闭煤气、关门锁门等的习惯，省得事后再去检查。
